docs/user-guide/configure-zowe-zosmf-workflow.md

```diff
@@ -59,17 +59,70 @@ After you register the workflow definition file, perform the following steps to
+    - **mvs**: Copies the data sets that are provided with Zowe to custom data sets.
+    - **security**: Creates user IDs and security manager settings.
+    - **apfauth**: APF authorizes the LOADLIB that contains the modules that perform priviledged security calls on z/OS.
+    - **certificate**: Configures Zowe to use TLS certificates.
+    - **vsam**: Configures the VSAM files that help run the Zowe caching service for high availability (HA)
+    - **stc**: Configures the system to launch the Zowe started task.
+
+    If you execute the init step again, either manually delete failed artifacts that are created from previous init steps or edit the step by adding the `--allow-overwritten` tag to the init command.
 
     **Example: Command that re-runs init**
 
     ```
     zwe init -c '/path/zowe.yaml' --allow-overwritten
     ```
 
-After you execute each step, the step is marked as Complete. After completing the workflow execution, you can view the Zowe started task.
+After you execute each step, the step is marked as complete. After completing the workflow execution, you can view the Zowe started task.
+
+## Execute the configuration workflow
+
+You can use the following methods to execute the configuration workflow:
+
+- Directly from a PSWI during deployment
+- From a deployed software instance (SI)
+- From the Workflows tab in the z/OSMF web UI
+
+### TODO Execute workflow from PSWI
+
+<!-- TODO `Workflow load` - what you had there before the change. It was unclear what exactly ""Workflow load"" means. I changed it to `Execute workflow` and also modified the heading a bit -->
+
+In the PSWI deployment phase, you are presented with the checklist that helps guide you during the deployment process.
+
+<img src="../images/zosmf/perform-workflows.png" alt="Workflow ZWECRECR" style="height: 300px; width:500px;"/> <br />
+
+The **perform workflows** step enables you to run either all attached workflows or just the mandatory one — the post-deployment workflow for mounting.
+
+### TODO Execute workflow from software instance
+
+<!-- TODO Same as above -->
+
+Software instance is created after PSWI deployment is complete. Execute a workflow from an SI.
+
+**Follow these steps**:
+
+1. Log in to z/OSMF.
+2. Select the **Software Management** panel.
+3. In the displayed table, select **Software Instances**.
+4. Select the checkbox next to the **Software Instance Name** column for the instance you want to execute the workflow against.
+5. Select the **Perform Workflows** option from the **Actions** menu.
+
+   The **Software Management Software Instances Perform Workflows** dialog opens.
+
+6. Select the **Create Workflow** option from the **Actions** menu.
+7. In the displayed table, click on the name of the workflow you want to execute.
+8. Click **OK**.
+
+    The **Workflows** tab with the previously selected workflow opens.
+
+9. TODO Continue with displayed steps same way as you would do from manual workflow registration.
```

Comment:
9. Execute displayed workflow the steps in order.

Reply:
I think I would rather go with something like "Follow the instructions on the page to execute the workflow steps". Do you think it's reasonable?

Reply:
issue is, there are no instruction displayed. We could either simplify just to: Execute the workflow steps
or rewrite what is written exactly for manual workflow execution, what is already in doc page

Execute the steps in order. Perform the following steps to execute each step individually:

a. Double-click the title of the step.

b. Select the Perform tab.

c. Review the step contents and update the input values as required.

d. Select Next.

Repeat the previous two steps to complete all items until the option Finish is available.  
![image](https://user-images.githubusercontent.com/45940302/172308852-27335105-b287-4372-b002-3867d9516711.png)